aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -378,15 +413,21 @@ class QConv1dPackWeightInt8 final {
 };
 
 TORCH_LIBRARY_IMPL(quantized, QuantizedCPU, m) {
+  // Conv
   // conv_prepack is deprecated, please use conv2d_prepack for 2D conv.
   m.impl("conv_prepack", TORCH_FN(QConvPackWeightInt8<2>::run_conv));
   m.impl("conv1d_prepack", TORCH_FN(QConv1dPackWeightInt8::run_conv));
   m.impl("conv2d_prepack", TORCH_FN(QConvPackWeightInt8<2>::run_conv));
   m.impl("conv3d_prepack", TORCH_FN(QConvPackWeightInt8<3>::run_conv));
+  // ConvTranspose
+  m.impl("conv_transpose2d_prepack", TORCH_FN(QConvPackWeightInt8<2>::run_deconv));
 }
 
 TORCH_LIBRARY_IMPL(_quantized, QuantizedCPU, m) {
+  // Conv
   m.impl("conv2d_prepack", TORCH_FN(QConvPackWeightInt8<2>::run_conv));
+  // ConvTranspose
+  m.impl("conv_transpose2d_prepack", TORCH_FN(QConvPackWeightInt8<2>::run_deconv));
```

Comment:
just curious, what's the reason to add this twice (also on line 423)?

Reply:
If I understand it correctly, the `_` prefix is still used, and will be removed once the ops are migrated to mobile. However, before this is done, it should be registered for both (I think): https://github.com/pytorch/pytorch/issues/36510

Reply:
I wonder if we still need to do it? @jerryzh168

Reply:
maybe let's follow up on if we need this?

Reply:
I am fine either way -- the only concern is that if removed it will break the consistency with respect to the `conv_Xd` methods.